packages/dev/loaders/src/glTF/2.0/Extensions/KHR_interactivity.ts

```diff
@@ -34,36 +40,138 @@ export class KHR_interactivity implements IGLTFLoaderExtension {
      */
     public enabled: boolean;
 
-    private _pathConverter?: InteractivityPathToObjectConverter;
+    private _pathConverter?: GLTFPathToObjectConverter<any, any, any>;
 
     /**
      * @internal
      * @param _loader
      */
     constructor(private _loader: GLTFLoader) {
         this.enabled = this._loader.isExtensionUsed(NAME);
-        this._pathConverter = new InteractivityPathToObjectConverter(this._loader.gltf);
+        this._pathConverter = getPathToObjectConverter(this._loader.gltf);
+        // avoid starting animations automatically.
+        _loader.parent.animationStartMode = GLTFLoaderAnimationStartMode.NONE;

```

Comment:
Hmm, not sure this will work right. What happens if someone sets this mode manually?

Reply:
We have to force this, according to the specs. What is the best way to be 100% sure no animation starts?

Reply:
One possibility is to add another flag (probably internal to the loader and loader extensions). Another is to make the loader function have an extensibility point and make it not respect the `animationStartMode` if interactivity is enabled.

Reply:
added a flag on the loader to skip `startAnimaion`